configuration/jsonkeyvalues/newEpisodesOfDiseasesReportConfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "report.newEpisodesOfDiseases.diagnosisList.conceptSet" : "3c93696a-64a3-4165-8d2f-93f725451ee6",
+    "report.newEpisodesOfDiseases.questions.conceptSet" : "46e49f70-e78f-4ef5-b7fe-1020d13f7fdb",
+    "report.newEpisodesOfDiseases.allDiagnoses.conceptSet" : "07058490-d3e5-46ef-bccc-d552ab7ce642",
```

Comment:
Let's try to make this support Set of Sets so that we can directly point to the existing Bahmni concept set:
Diagnosis Set of Sets: `dbc2ecdb-4e14-11e4-8a57-0800271c1b75`